recipes/UniFunc/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "unifunc" %}
+{% set version = "1.2.1" %}
+
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/PedroMTQ/UniFunc/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 9eaa196505dabe471b3ac790f3e84770919f53728de2f6f0ecbbb550812fde97
```

Comment:
This needs to be updated to the newest one.

Reply:
done